### PR TITLE
RA-1814:Unnecessary imports should be removed

### PR DESCRIPTION
**Pull request by Wandji69**
https://issues.openmrs.org/browse/RA-1814

### CODE REVIEW

**Comment by sacull** [comment]
@Wandji69 In the description of the PR, it is worth providing a link to the ticket in JIRA. The person who will give opinions on the changes will know what ticket this PR applies to.

File: omod/src/test/java/org/openmrs/module/referenceapplication/page/controller/HomePageControllerTest.java [hunk 1]
```diff
@@ -42,12 +38,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```
Comment (sacull) [inline omod/src/test/java/org/openmrs/module/referenceapplication/page/controller/HomePageControllerTest.java:41]: Try not to use wildcard import. Ref: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Imports

**Review by sacull — CHANGES_REQUESTED** [review]

**Comment by sacull** [comment]
You have not forgotten about this import: https://github.com/openmrs/openmrs-module-referenceapplication/blob/referenceapplication-2.10.0/omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/HomePageController.java#L23 ;)

**Review by sacull — APPROVED** [review]
It looks good to me now. ;)

**Comment by Wandji69** [comment]
@sacull Thanks for reviewing.

**Comment by sacull** [comment]
@Wandji69 You are probably ready to leave the introductory tickets and to take care of the more complex ones. ;)

**Comment by Wandji69** [comment]
> @Wandji69 You are probably ready to leave the introductory tickets and to take care of the more complex ones. ;)

Okay, thanks. I will round up with the tickets I have at hand before moving forward.

**Pull request merged**